dags/openshift_nightlies/config/install/rosa/ovn-small.json

```diff
@@ -12,5 +12,6 @@
     "openshift_network_type": "OVNKubernetes",
     "openshift_worker_instance_type": "m5.2xlarge",
     "machineset_metadata_label_prefix": "machine.openshift.io",
-    "openshift_workload_node_instance_type": null
+    "openshift_workload_node_instance_type": "m5.2xlarge",
+    "enable_spot_workers": false
```

Comment:
Please create a new file: `dags/openshift_nightlies/config/install/rosa/ovn-small-spot.json`
And move the key/values to it.

Reply:
if it is the case, I need to add this file to the manifest.yml to read properties of this file.

Reply:
> if it is the case, I need to add this file to the manifest.yml to read properties of this file.

Please update accordingly

Reply:
Done